x-pack/plugins/security/public/plugin.ts

```diff
@@ -21,6 +21,7 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
     anonymousPaths.register('/login');
     anonymousPaths.register('/logout');
     anonymousPaths.register('/logged_out');
+    anonymousPaths.register('/status');
```

Comment:
Two comments on this:
1. The status page may require authentication unless the `status.allowAnonymous` setting is used
2. I already added a NP `status_page` plugin in #49855 (recently merged, probably after you started working on this PR), see [`plugin.ts`](https://github.com/elastic/kibana/blob/7cb55131b5d476c1e8665e3cbeceb41bc89908db/src/plugins/status_page/public/plugin.ts#L24-L30), so this code is redundant now :)

TL;DR I believe you can remove this line now!

Reply:
```
if (isStatusPageAnonymous) {
      core.http.anonymousPaths.register('/status');
    }
```
This is indeed better. Thanks a lot, will remove